docs/how-to/install-slice.rst

```diff
@@ -0,0 +1,96 @@
+How-to install from my own custom slice definitions?
+****************************************************
+
+First, you need to clone this repository:
+
+.. code-block:: sh
+
+    $ git clone https://github.com/canonical/chisel-releases/
+
+This repository is the database of slices definitions file. After that, you need to put your openssl.yaml file into the chisel-releases/slices folder.
+
+| Now we will install our openssl slice into our ROCK'S, so to initialize a rock you need to do:
+
+.. code-block:: sh
+
+    $ rockcraft init
+
+after this command your folder looks like this:
+
+.. code-block:: text
+
+    -chisel-releases/
+    -rockcraft.yaml
+
+Now you need to copy and paste this code into the rockcraft file:
+
+.. code-block:: yaml
+
+    name: ourslice
+    base: bare
+    build_base: "ubuntu:22.04"
+    version: '0.0.1'
+    summary: Install the slice
+    description: |
+        Slice install tutorial
+    license: GPL-3.0
+    platforms:
+        amd64:
+    parts:
+        build-context:
+            plugin: dump
+            source: chisel-releases/
+            source-type: local
+            organize:
+                '*': chisel-releases/
+
+        installslice:
+            plugin: go
+            source: https://github.com/canonical/chisel
```

Comment:
do you actually need to compile Chisel? it is already being shipped with the Rockcraft Snap, so I wonder whether it could be already available at build time...

Reply:
I don't think it's available at the build name, do you have the command for testing?

Reply:
@tigarmo since utilities like Skopeo and Chisel are shipped with the Rockcraft snap, is there a way to use them from within a part?

Reply:
Done, only need to call chisel. I have updated the yaml file